Podcast/PlayerEpisodeDetailView.swift

```diff
@@ -10,99 +10,112 @@ import UIKit
 
 class PlayerEpisodeDetailView: UIView {
     
+    var expandedArtwork: Bool
+    
     var episodeArtworkImageView: UIImageView!
-    var seriesTitleLabel: UILabel!
     var episodeTitleLabel: UILabel!
     var dateLabel: UILabel!
-    var descriptionLabel: UILabel!
-    var recommendButton: UIButton!
+    var descriptionTextView: UITextView!
+    var seeMoreButton: UIButton!
+    
+    let marginSpacing: CGFloat = 18
+    let artworkY: CGFloat = 10
+    
+    let artworkLargeDimension: CGSize = CGSize(width: 300, height: 300)
+    let artworkSmallDimension: CGSize = CGSize(width: 60, height: 60)
     
-    let leftMargin: CGFloat = 24
-    let artworkDimension: CGFloat = 45.5
-    let episodeTitleYInset: CGFloat = 81.5
-    let dateLabelYSpacing: CGFloat = 12
-    let descriptionLabelSpacing: CGFloat = 24
+    let episodeTitleYInset: CGFloat = 13
+    let dateLabelYSpacing: CGFloat = 8
+    let descriptionTextViewSpacing: CGFloat = 8
     let recommendButtonSpacing: CGFloat = 22.5
     let bottomSpacing: CGFloat = 28.5
     
     override init(frame: CGRect) {
+        expandedArtwork = true
         super.init(frame: frame)
-        backgroundColor = .white
+        backgroundColor = UIColor.colorFromCode(0xf4f4f7)
         
         episodeArtworkImageView = UIImageView(frame: .zero)
```

Comment:
Use ImageView... don't init to zero tho! Initialize to the frame it should be!! Then call setImageAsynchronously on the imageview